.github/workflows/apps.yml

```diff
@@ -62,15 +62,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-    - name: Get Latest Release Tag
-      run: |
-        WARPX_TAG=$(wget https://github.com/ECP-WarpX/WarpX/releases/latest 2>&1 | grep Location: | awk '{print $2}' | awk -F/ '{print $NF}')
```

Comment:
Discussed: update to find the latest commit that has the words `AMReX`, `Update` (case insensitive and arbitrary order) in it
https://github.com/ECP-WarpX/WarpX/pull/5172
or Release
https://github.com/ECP-WarpX/WarpX/pull/5125
or is the latest tag, whatever is newer.